spec/system/managing_topics_spec.rb

```diff
@@ -32,7 +33,7 @@
     visit topics_path
     click_on('What is your favourite colour')
     expect(current_path).to eq topic_path(topic)
-    expect(page).to have_content('Topic:')
+    expect(page).to have_content(topic.user.username)
```

Comment:
Generally, it is more robust to expect the actual content rather than the result of calling methods. For example, what if for some reason the `topic.user.username` evaluated to  `nil`, then this test would pass even if you weren't presenting the username on the page.